sdk/storage/AzureStorageBlob/Source/DataStructures/TransferCollection.swift

```diff
@@ -82,11 +82,11 @@ public struct TransferCollection {
         blobName _: String? = nil,
         localUrl _: LocalURL? = nil,
         state _: TransferState? = nil
-    ) -> [BlobTransfer] {
-        return all.filter { match($0) }
+    ) -> TransferCollection {
```

Comment:
I like this. Now I can filter, get the collection, and then cancel/delete/whatever the whole result of the filter.